build.gradle

```diff
@@ -3,13 +3,31 @@ description = 'Opencensus Proto'
 apply plugin: 'idea'
 apply plugin: 'java'
 apply plugin: 'com.google.protobuf'
+apply plugin: 'maven'
+apply plugin: "signing"
+
+group = "io.opencensus"
+version = "0.0.1-SNAPSHOT" // CURRENT_OPENCENSUS_VERSION
+
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
 
 repositories {
     maven { url "https://plugins.gradle.org/m2/" }
 }
 
+jar.manifest {
+    attributes('Implementation-Title': name,
+            'Implementation-Version': version,
+            'Built-By': System.getProperty('user.name'),
+            'Built-JDK': System.getProperty('java.version'),
+            'Source-Compatibility': sourceCompatibility,
+            'Target-Compatibility': targetCompatibility)
+}
+
 def protobufVersion = '3.5.1'
 def protocVersion = '3.5.1'
+def grpcVersion = "1.15.0" // CURRENT_GRPC_VERSION
```

Comment:
Are we actually pinning dependencies to the latest gRPC-java version?

Reply:
Chatted offline, we'll use the same gRPC version with OpenCensus-Java, which is [1.14.0](https://github.com/census-instrumentation/opencensus-java/blob/c0e0b225a5230036b3dc09134f94eece81d3f96d/build.gradle#L156).